internal/cli/atlas/kubernetes/kubernetes.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kubernetes
+
+import (
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli/atlas/kubernetes/config"
+	"github.com/spf13/cobra"
+)
+
+func Builder() *cobra.Command {
+	const use = "kubernetes"
+	cmd := &cobra.Command{
+		Use:     use,
+		Aliases: cli.GenerateAliases(use),
+		Short:   "Manage kubernetes resources.",
+		Long:    "The kubernetes command provides access to kubernetes features of the atlas.",
+	}
+
+	cmd.AddCommand(config.Builder())
+
+	return cmd
```

Comment:
Can you add a `TestBuilder`? here is an example https://github.com/mongodb/mongodb-atlas-cli/blob/master/internal/cli/atlas/dbusers/dbusers_test.go

Reply:
Thanks! I added one